src/plugins/curator/components/teaser-picker-field/SortableList/index.jsx

```diff
@@ -0,0 +1,177 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Sortable from 'sortablejs';
+import { connect } from 'react-redux';
+import damUrl from '@triniti/cms/plugins/dam/utils/damUrl';
+import pbjUrl from '@gdbots/pbjx/pbjUrl';
+import {
+  Button,
+  ButtonToolbar,
+  FormGroup,
+  Icon,
+  Label,
+  Media,
+  Row,
+} from '@triniti/admin-ui-plugin/components';
+
+import selector from './selector';
+
+class SortableList extends React.Component {
+  static propTypes = {
+    fields: PropTypes.object.isRequired, // eslint-disable-line react/forbid-prop-types
+    getNode: PropTypes.func.isRequired,
+    isMulti: PropTypes.bool,
+    readOnly: PropTypes.bool,
+  };
+
+  static defaultProps = {
+    isMulti: true,
+    readOnly: false,
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.handleMoveDown = this.handleMoveDown.bind(this);
+    this.handleMoveUp = this.handleMoveUp.bind(this);
+  }
+
+  componentDidMount() {
+    const { fields } = this.props;
+    Sortable.create(this.sortableListRef, {
+      animation: 150,
+      chosenClass: 'active',
+      draggable: '.sortable-articles',
+      filter: '.js-remove',
+      handle: '.sort-handle',
+      onFilter: (evt) => {
+        fields.remove(evt.oldIndex);
+      },
+      onUpdate: (evt) => {
+        fields.move(evt.oldIndex, evt.newIndex);
+      },
+    });
+  }
+
+  handleMoveDown(index) {
+    const { fields } = this.props;
+    if (index === fields.length - 1) {
+      return;
+    }
+    fields.move(index, index + 1);
+  }
+
+  handleMoveUp(index) {
+    const { fields } = this.props;
+    if (index === 0) {
+      return;
+    }
+    fields.move(index, index - 1);
+  }
+
+  render() {
+    const { fields, getNode, isMulti, readOnly } = this.props;
+    let label = isMulti ? 'order teasers' : 'selected teaser';
+    if (readOnly) {
+      label = isMulti ? 'related teasers' : 'related teaser';
+    }
+
+    return (
+      <FormGroup>
+        <Label className="mb-2">{label}</Label>
+        <Row>
+          <div ref={(el) => { this.sortableListRef = el; }} className="col">
+            {fields.getAll().map((articleRef, i) => {
+              const article = getNode(articleRef);
```

Comment:
@albertgcsula this has a bunch of references to "article" or "articles" but these are teasers.

Reply:
@gdbrown updated.